src/apis/reviews/query.ts

```diff
@@ -0,0 +1,25 @@
+import {useInfiniteQuery, useMutation} from '@tanstack/react-query';
+import {getReveiewLists, postReviewReply} from './client';
+import {ReviewListsRequest, ReviewReplyCreateRequest} from './model';
+
+export const useReviewList = (marketId: number) => {
+  return useInfiniteQuery({
+    queryKey: ['marketList', marketId],
+    queryFn: ({pageParam = 0}) =>
+      // FIXME: 페이징 단위 size
+      getReveiewLists({cursorId: pageParam, size: 5, marketId}),
+    initialPageParam: 0,
+    getNextPageParam: lastPage => {
+      return lastPage?.hasNext ? lastPage.reviews.length : undefined;
+    },
```

Comment:
```suggestion
    getNextPageParam: lastPage => {
      return lastPage?.hasNext ? lastPage.reviews[lastPage.reviews.length - 1].id : undefined;
    },
```

커서 기반이라서 이렇게 해야지 잘 될 거에요